### ISSUE TITLE
Any wants to release the code for MASS?

### ISSUE THREAD

**Issue by circlePi** [xutaatmicrosoftdotcom/MASS#1]
Any wants to release code and pre-trained models for the paper "MASS: Masked Sequence to Sequence Pre-training for Language Generation"?
